physionet-django/events/templates/events/event_participation.html

```diff
@@ -0,0 +1,21 @@
+<div class="container">
+  <h5>Event Registration</h5>
+  {% if registration_allowed %}
+    <strong><p>Click the button below to join the waiting list. You will receive a notification when your registration is confirmed.</p></strong>
+
+    <form  method="post">
+      {% csrf_token %}    
+      <button type="submit" class="btn btn-sm btn-primary" name="confirm_registration" value="confirm" role="button">Join waitlist</button>
+    </form>
+  {% else %}
+    <p>{{registration_error_message}}</p>
```

Comment:
I think the template syntax should have a space between curly brackets and the variable (i.e. `{{ registration_error_message }}` not `{{registration_error_message}}`

Reply:
Ah yes, i missed this